File: app/src/main/java/com/bot/man/api/APIWrapper.java
package com.bot.man.api;

import com.bot.man.model.data.Response;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface APIWrapper {
	@GET("getUpdates")
	Call<Response> fetchGetUpdates();

	@GET("getUpdates?timeout=100")
	Call<Response> getUpdate(@Query("offset") Integer offset);
}

File: app/src/main/java/com/bot/man/api/APIWrapper.kt
package com.bot.man.api

import com.bot.man.model.data.Response
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface APIWrapper {
    @GET("getUpdates")
    fun fetchGetUpdates(): Call<Response?>?

    @GET("getUpdates?timeout=100")
    fun getUpdate(@Query("offset")
                  offset: Int?): Call<Response?>?

    @GET("sendMessage")
    fun sendMessage(
            @Query("text") text: String?, @Query("chat_id")
            chat_id: Int?): Call<Response?>?
}